README.md

```diff
@@ -38,6 +38,8 @@ Although in the global scope, it is not available until after the `deviceready`
         console.log(navigator.globalization);
     }
 
+:warning: Check issues on the [JIRA issue tracker](https://issues.apache.org/jira/browse/CB-7790?filter=-1&jql=project%20%3D%20CB%20AND%20status%20in%20(Open%2C%20%22In%20Progress%22%2C%20Reopened)%20AND%20resolution%20%3D%20Unresolved%20AND%20component%20%3D%20%22Plugin%20Globalization%22%20ORDER%20BY%20priority%20DESC%2C%20summary%20ASC%2C%20updatedDate%20DESC)
```

Comment:
The URL is wrong, it always takes the user to CB-7790

I'm not JIRA query builder expert but, I build this one and I think it does what you want:
https://issues.apache.org/jira/issues/?jql=project%20%3D%20CB%20AND%20status%20in%20(Open%2C%20%22In%20Progress%22%2C%20Reopened)%20AND%20resolution%20%3D%20Unresolved%20AND%20component%20%3D%20%22Plugin%20Globalization%22%20ORDER%20BY%20priority%20DESC%2C%20summary%20ASC%2C%20updatedDate%20DESC